httpx/_transports/wsgi.py

```diff
@@ -80,6 +80,12 @@ class WSGITransport(BaseTransport):
     ```
     """
 
+    app: typing.Callable[..., typing.Any]
+    raise_app_exceptions: bool
+    script_name: str
+    remote_addr: str
+    wsgi_errors: typing.Optional[typing.TextIO]
```

Comment:
We don't *really* want these to be public API.

The `HTTPTransport` class has it's implementation attributes use a private namespace, but the `WSGITransport` and `ASGITransport` classes currently don't.